pandas_datareader/tests/test_google_options.py

```diff
@@ -0,0 +1,39 @@
+import os
+from datetime import datetime
+
+import numpy as np
+from pandas import DataFrame, Timestamp
+
+import nose
+import pandas.util.testing as tm
+from pandas_datareader.tests._utils import _skip_if_no_lxml
+
+import pandas_datareader.data as web
+from pandas_datareader._utils import RemoteDataError
+
+
+class TestGoogleOptions(tm.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestGoogleOptions, cls).setUpClass()
+
+        # goog has monthlies
+        cls.goog = web.Options('GOOG', 'google')
+
+    def test_get_options_data(self):
+        try:
+            options = self.goog.get_options_data(expiry=self.goog.expiry_dates[0])
+        except RemoteDataError as e:  # pragma: no cover
+            raise nose.SkipTest(e)
+        self.assertTrue(len(options) > 10)
```

Comment:
Pls add a test to check whether data has an expected format, compare columns etc.